app/qml/pages/MainMenuPage.qml

```diff
@@ -72,17 +72,68 @@ Page {
       
             Label {
                 id: batteryLabel
-                anchors.topMargin: units.gu(1)
                 text: curWatchConnected ? batteryLevel + ("%") : i18n.tr("unkown")
             }
         }
     }
+    
+    // Divider
+    Rectangle {
+      id: dividerRect
+      anchors.top: statusRowLayout.bottom
+      width: parent.width      
+      height: 1
+      color : "grey"
+    }
+    
+    ScrollView {
+        id: scrollView
+        width: parent.width
+        height: parent.height
+        anchors.top: dividerRect.bottom
+        
+        contentItem: Column {
+            width: scrollView.width
+            
+            ListItem {
+              
+                ListItemLayout {
+                    title.text: i18n.tr("Synchronize time")
+                    Icon {
+                        name: "clock-app-symbolic"
+                        color: "black"
+                        width: units.gu(3)
+                        height: units.gu(3)
+                        SlotsLayout.position: SlotsLayout.Leading
+                    }
+
+                    Switch {
+                        id: timeSyncSwitch
+                        SlotsLayout.position: SlotsLayout.Trailing
+                        onCheckedChanged: settings.timeSync = checked
+                    }
+                }
+            }
+        }
+    }
+
+    Binding {
+        target: timeSyncSwitch
+        property: "checked"
+        value: settings.timeSync
+    }
 
     Timer {
         interval: 1000
         running: curWatchConnected
         repeat: true
         onTriggered: batteryLevel = root.watch.batteryLevel
     }
-    
+
+    Timer {
+        interval: 0
+        running: root.watch.timeServiceReady && settings.timeSync
```

Comment:
Why do you use a timer with 0 interval and repeat set to false?
Couldn't you use a Connections instead https://doc.qt.io/qt-5/qml-qtqml-connections.html ?
I imagine that if you'd set target to root.watch and bind to onTimeServiceReadyChanged, you could write a simple slot that does: if(root.watch.timeServiceReady && settings.timeSync) root.watch.setTime(Date())
This would be clearer since you'd eliminate the whole "timer" semantic which doesn't make much sense to me here

Reply:
I couldn't think of a better solution. Thank you very much for your help!

Reply:
done